nodejs/week3/app.js

```diff
@@ -0,0 +1,49 @@
+const knex = require("knex")({
+    client: "mysql2",
+    connection: {
+        host: process.env.DB_HOST || "127.0.0.1",
+        port: process.env.DB_PORT || 3306,
+        user: process.env.DB_USER || "root",
+        password: process.env.DB_PASSWORD || "root",
+        database: process.env.DB_NAME || "hyf_node_week3_warmup",
+        multipleStatements: true,
+    },
```

Comment:
I'd really recommend not putting default values in place like this. Better have an error if something is missing than roll with half-done configurations.